app/lib/features/activities/pages/activities_page.dart

```diff
@@ -80,7 +80,9 @@ class ActivitiesPage extends ConsumerWidget {
           showSectionBg: false,
           isShowSeeAllButton: false,
         ),
-        InvitationCard(invitation: invitations.first),
+        InvitationWidget(
```

Comment:
```suggestion
        InvitationItemWidget(
```

User proper class and widget naming based on the use-case.